pallets/did/src/lib.rs

```diff
@@ -248,30 +248,44 @@ pub trait DidOperation<DidIdentifier>: Encode {
 /// * The optional attestation key to use
 /// * The optional delegation key to use
 /// * The optional endpoint URL pointing to the DID service endpoints
-#[derive(Clone, Decode, Debug, Encode, PartialEq)]
-pub struct DidCreationOperation<DidIdentifier>
-where
-	DidIdentifier: Parameter + Encode + Decode + Debug,
-{
-	did: DidIdentifier,
+#[derive(Clone, Decode, Encode, PartialEq)]
+pub struct DidCreationOperation<T: Config> {
```

Comment:
i would rather keep the `DidIdentifier` but I'm open to talk about it. 🙂

Reply:
Same as above.